tests/python/unittest/test_tir_schedule_tensorize_mfma.py

```diff
@@ -0,0 +1,322 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=missing-docstring
+import tvm
+from tvm import te
+from tvm.tir.tensor_intrin.rocm import (
+    shared_16x4_to_local_64x1_layout_A,
```

Comment:
Could we consider change it to 
```python
from tvm.tir.tensor_intrin import rocm
# or
from tvm.tir.tensor_intrin.rocm import *
```

Reply:
it can be, but this part of import is aligned with [test_tir_schedule_tensorize_ldmatrix_mma.py](https://github.com/apache/tvm/blob/main/tests/python/unittest/test_tir_schedule_tensorize_ldmatrix_mma.py)